js/index.js

```diff
@@ -40,3 +40,97 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let ctaImg = document.getElementById('cta-img');
+ctaImg.setAttribute('src', siteContent["cta"]["img-src"])
+ 
+let middleImg = document.getElementById("middle-img");
+middleImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+
+let navAnchors = document.querySelectorAll("a");
+ 
+navAnchors.forEach(function(item, index, array){
+ item.innerText=siteContent['nav'][`nav-item-${index + 1}`];
+ item.style.color="green";
+});
+ 
+const newAnchor1 = document.createElement("a");
+ 
+const newAnchor2 = document.createElement("a");
+ 
+newAnchor1.textContent = "Come Here";
+newAnchor2.textContent = "Go Away";
+ 
+newAnchor1.style.color = "slateblue";
+newAnchor2.style.color = "mediumorchid";
+ 
+let navBar = document.querySelector("nav");
+navBar.appendChild(newAnchor2);
+ 
+navBar.prepend(newAnchor1);
+
+const topH4 = document.querySelectorAll(".top-content .text-content h4");
+ 
+topH4[0].style.color = 'mediumorchid';
+topH4[1].style.color = 'slateblue';
+topH4[0].innerText = siteContent['main-content']['features-h4'];
+topH4[1].innerText = siteContent['main-content']['about-h4'];
+ 
+ 
+ 
+const topPar = document.querySelectorAll('.top-content .text-content p');
+ 
+topPar[0].innerText = siteContent['main-content']['features-content'];
+topPar[1].innerText = siteContent['main-content']['about-content'];
+
+
+const bottomh4 = document.querySelectorAll('.bottom-content .text-content h4');
+ 
+bottomh4[0].innerText = siteContent['main-content']['services-h4'];
+bottomh4[1].innerText = siteContent['main-content']['product-h4'];
+bottomh4[2].innerText = siteContent['main-content']['vision-h4'];
+ 
+const bottomPars = document.querySelectorAll('.bottom-content .text-content p');
+ 
+bottomPars[0].innerText = siteContent['main-content']['services-content'];
+bottomPars[1].innerText = siteContent['main-content']['services-content'];
+bottomPars[2].innerText = siteContent['main-content']['services-content'];
+ 
+
+
+
+const cta1 = document.querySelector('.cta-text h1');
+cta1.style.color = "mediumorchid";
+let ctaArray = siteContent['cta']['h1'].split(" ");
+ 
+ctaArray.splice(1,0, "\r\n");
+ctaArray.splice(3,0, "\r\n");
+ 
+cta1.innerText = ctaArray.join(" ");
```

Comment:
I like this bit of code. I'm interested to see the output and hear the breakdown. Clean way to approach that "DOM is Awesome" text